canvas/migrations/0010_merge_20210730_1543.py

```diff
@@ -0,0 +1,14 @@
+# Generated by Django 3.0.14 on 2021-07-30 22:43
```

Comment:
Can this and the above migration be combined after merging with the latest version on master.